InWorldz/InWorldz.ApplicationPlugins/ChatLog/ChatLogModule.cs

```diff
@@ -110,7 +110,7 @@ public void RegionLoaded(Scene scene)
         void EventManager_OnChatToClient(string message, UUID fromID, UUID toID, UUID regionID, 
             uint timestamp, ChatToClientType type)
         {
-            if (message == "" || toID == fromID) return;
+            if (String.IsNullOrEmpty(message) || toID == fromID) return;
```

Comment:
Semantic change: null messages now cause early return instead of the ChatMessageLog instance getting it.  Not sure nulls even make it through here, as sending a null message doesn't make sense.